spec/02-integration/09-hybrid_mode/01-sync_spec.lua

```diff
@@ -16,8 +16,16 @@ local CLUSTERING_SYNC_STATUS = require("kong.constants").CLUSTERING_SYNC_STATUS
 local KEY_AUTH_PLUGIN
 
 
+local confs = {
+  wrpc = "spec/fixtures/custom_nginx.template",
+  json = "spec/fixtures/json_nginx.template",
```

Comment:
Should this be written to `/tmp/` instead? Otherwise this will create additional files inside  `spec/fixtures/` that might be accidentally committed (or add `.gitignore`).

Also, we can change the filename to `custom_nginx_no_wrpc.template` to avoid consuming people if they seen this file under `fixtures` during development.

Reply:
Also, consider making this a function in `helper` instead, this seems to be very duplicative between 3 files right now.

You can define a function `helpers.get_clustering_protocols` that returns this table and sets up the patched file at the same time.

Reply:
done

Reply:
this sidesteps the need for the user to override the protocol